components/form/form.js

```diff
@@ -34,6 +34,16 @@ class Form extends LocalizeCoreElement(LitElement) {
 			 * @type {boolean}
 			 */
 			trackChanges: { type: Boolean, attribute: 'track-changes', reflect: true },
+			/**
+			 * Indicates that the form should hide the error summary alert.
+			 * @type {boolean}
+			 */
+			hideErrorSummary: { type: Boolean, attribute: 'hide-error-summary' },
```

Comment:
I think we can just ditch `hideErrorSummary` for now since primary/secondary no longer needs it? We can always add it back later if it's needed for something else.